src/qibocal/protocols/ramsey/utils.py

```diff
@@ -90,13 +90,17 @@ def fitting(x: list, y: list, errors: list = None) -> list:
 
     period = fallback_period(guess_period(x, y))
     omega = 2 * np.pi / period
+    idx = np.argmin(x)
```

Comment:
Isn't this equivalent to
```suggestion
    idx = 0
```
?